sky/engine/core/events/VelocityTracker.idl

```diff
@@ -0,0 +1,11 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[
+  Constructor(),
+] interface VelocityTracker {
+  void addPosition(long timeStamp, long pointerId, float x, float y);
```

Comment:
I discard all of the pointer data when the last pointer goes up. Only a limited number of positions are buffered (by VelocityTracker) per pointer (20 I think) so I thought that would be OK.